sensors/06-list-datasets-by-plot.Rmd

```diff
@@ -55,28 +56,30 @@ The geostreams API can be used to get a list of datasets that overlap a
 specific plot boundary and, optionally, limited by a time range. Iterating 
 over the datasets allows the paths to all the files to be extracted.
 
-```
+```{python}
 sensor = 'Thermal IR GeoTIFFs Datasets'
 sitename = 'MAC Field Scanner Season 1 Field Plot 101 W'
 datasets = get_file_listing(None, url, key, sensor, sitename)
 files = extract_file_paths(datasets)
 ```
 
 Datasets can be further filtered using the *since* and *until* parameters
-of get\_file\_listing with a date string.
+of `get_file_listing` with a date string.
 
-```
+```{python}
 dataset = get_file_listing(None, url, key, sensor, sitename, 
         since='2016-06-01', until='2016-06-10')
 ```
 
 
 # Alternative method
+
 The following method demonstrates the same approach using the Clowder API. This
 approach is useful for understanding the data layout and when the Python
 terrautils package is not available.
 
 ## Finding plot ID
+
```

Comment:
@jterstriep How are these commands run? I get syntax errors in both bash and python